### ISSUE TITLE
Questions Regarding Table 2 in your paper: Evaluation Setup and Data Split

### ISSUE THREAD

**Issue by iGuoYanjun** [snap-research/ac3d#16]
Thank you so much for your great work and for open-sourcing the code!

I have some questions regarding the qualitative results shown in Table 2 of your paper:

(1) Is Table 2 based on camera pose control applied to the I2V baseline? 
	
(2) I’m also curious about the data used for metric calculation. Could you kindly provide more details in order to let me better understand  how the metrics were computed in table 2.
	
Specifically, for RE10K, could you clarify which subset of the data was used to compute the reported metrics?
	
For example, in the CameraCtrl paper, they mention sampling 1,000 examples randomly from the RE10K test set. Did your evaluation also follow this protocol, or did you use the official test split directly?

Thanks again for your contribution and looking forward to your clarification!

**Comment by sherwinbahmani** [comment]
Hi,

(1) We use text-to-video for the metrics.
(2) We use the full test split. The evaluation is a bit different from CameraCtrl, since in CameraCtrl there was no aligment process between input and target trajectories. But we normalized the scales between input and groundtruth, since COLMAP gives you scale ambiguity which can distort evaluations.

**Comment by iGuoYanjun** [comment]
Thank you so much for your prompt response!

I have a quick follow-up question. Are both motion-ctrl and CameraCtrl results in Table 2 based on AnimateDiff?

I noticed that your paper mentions using a your own video generation model. If I use the cogvideo2b model provided in this repository, would it be possible to reproduce results that are close to those reported in Table 2?

Thanks again for your clarification and kind support!

**Comment by sherwinbahmani** [comment]
We did the main comparisons on top of the same internal SnapVideo model. It is a bit unfair to compare between different base video models, as the new DiT models are much stronger and will always give better results. So we re-implemented the methods on top of the same base model.

One issue is that the evaluation pipeline used in the paper is integrated into the internal SnapVideo infrastructure. That's why I didn't open-source, since it would need to be ported first.

My suggestion is to have proper comparisons: Take your method and take whatever method you want to compare with. In the best case both methods should use the same video model and are trained with the same data. Then you take the same evaluation scripts and evaluate them from scratch. We provided some pre-trained checkpoints for CogVideoX. So I would re-evaluate them with the evaluation scripts and your method with the same scripts. Otherwise you run into issues that the evaluations are not consistent, since unfortunately, there is no public unified evaluation for all these benchmarks.

**Comment by iGuoYanjun** [comment]
Thank you for your detailed response — I really appreciate it. I also believe that fair comparison should **ideally** be based on the same baseline and training data. 

However, since both motion-ctrl and CameraCtrl are designed around a UNet-based architecture,  I was a bit unsure about the evaluation setup.

Since I am exactly not sure how large the performance gap is between your internal SnapVideo model and open-source CogvideX, I’m not sure what range of discrepancies would be considered reasonable — i.e., caused by differences in the base models — and what might instead indicate an issue with my own evaluation code.

By the way, I have to say — your analysis in the paper has been truly inspiring. Thank you again for your excellent work, and best wishes for your future research!

**Comment by sherwinbahmani** [comment]
There is some significant gap between SnapVideo and CogVideoX. The implementation here is more for an easy reference to build upon and modify. Unfortunately, the results from the paper and project page can not be achieved easily with CogVideoX as it is a weaker base video model. But there are newer video models like Wan 2.1, which can be used if CogVideoX is not good enough.

Try to re-evaluate all methods using your own evaluation code and report those numbers, I think this is the safest thing to do.

**Comment by iGuoYanjun** [comment]
Thanks again for your detailed reply!
